server/routes/TripRoute.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express')
+const controllers = require('../controllers/Controllers')
+
+class TripRoute {
+  constructor() {
+    this.router = express.Router()
+    this.router.get('', this.getAll.bind(this))
+    this.router.post('', this.create.bind(this))
+    this.router.patch('/:id', this.edit.bind(this))
+    this.router.delete('/:id', this.delete.bind(this))
+  }
+
+  initRoutes(apiRouter) {
+    apiRouter.use('/trip', this.router)
+  }
+
+  async getAll(req, res) {
+    try {
+      const response = await controllers.tripController.getAll()
+      res.status(200).json(response)
+    } catch (err) {
+      res.status(500).json({ error: err.toString() })
+    }
+  }
+
+  async create(req, res) {
+    try {
+      const newTrip = await controllers.tripController.createTrip(req.body)
+      res.status(201).json(newTrip)
+    } catch (err) {
+      res.status(500).json({ error: err.toString() })
+    }
+  }
+
+  async edit(req, res) {
```

Comment:
Should we keep this consistent with either CRUD terminology or the HTTP request types?

Reply:
hm, i havent thought about that. good idea